src/index.html

```diff
@@ -22,6 +22,317 @@
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button class="card__button">BUY</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacBook"
+          class="card__image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review-wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button class="card__button">BUY</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacBook"
+          class="card__image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review-wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button class="card__button">BUY</button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="MacBook"
+          class="card__image"
+        />
+
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__review-wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__review">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-label">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+        <button class="card__button">BUY</button>
+      </div>
```

Comment:
Consider using a loop or a template engine to generate these repetitive product card elements dynamically. This will make your code more maintainable and easier to update in the future.